src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -75,16 +70,6 @@ public void TestInitialize()
             this.serviceProvider.RegisterService(typeof(ISourceControlledFileSystem), sccFileSystem);
             this.serviceProvider.RegisterService(typeof(IProjectSystemHelper), this.projectSystemHelper);
 
-            projectToLanguageMapper = new Mock<IProjectToLanguageMapper>();
-            folderWorkspaceService = new Mock<IFolderWorkspaceService>();
-
-            var mefProjectToLanguageMapper =
-                ConfigurableComponentModel.CreateWithExports(
-                    MefTestHelpers.CreateExport<IProjectToLanguageMapper>(projectToLanguageMapper.Object),
-                    MefTestHelpers.CreateExport<IFolderWorkspaceService>(folderWorkspaceService.Object));
-
-            serviceProvider.RegisterService(typeof(SComponentModel), mefProjectToLanguageMapper);
```

Comment:
The class no longer MEF-imports either `IProjectToLanguageMapper` or `IFolderWorkspaceService`